integration/js/app/video_test/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "video_test",
+  "version": "1.0.0",
+  "description": "Mini Test App for running video Api integ tests",
+  "scripts": {
+    "deps": "cd ../../../.. && npm run build",
+    "build:fast": "tsc && webpack --config ./webpack.config.js",
+    "build": "npm run deps && npm install && npm run build:fast",
+    "start:fast": "npm run build:fast && node server.js",
+    "tsc": "tsc --listFiles",
+    "start": "npm run deps && npm install && npm run start:fast"
+  },
+  "devDependencies": {
+    "autoprefixer": "^9.6.4",
+    "css-loader": "^4.3.0",
+    "file-loader": "^6.1.0",
+    "html-webpack-inline-source-plugin": "^0.0.10",
+    "html-webpack-plugin": "^3.2.0",
+    "node-sass": "^4.14.1",
+    "postcss-loader": "^3.0.0",
+    "precss": "^4.0.0",
+    "sass": "^1.26.11",
+    "sass-loader": "^10.0.2",
+    "style-loader": "^1.2.1",
+    "ts-loader": "^8.0.4",
+    "typescript": "^4.0.0",
+    "webpack": "^4.44.2",
+    "webpack-cli": "^3.3.12"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "amazon-chime-sdk-js": "file:../../../..",
+    "aws-sdk": "^2.802.0",
+    "compression": "^1.7.4"
```

Comment:
Do we need compression? See third point in this issue https://github.com/aws/amazon-chime-sdk-js/issues/628

Reply:
No we dont, I remved the dependency. I'll create a seperate task to switch to using express for this server code